pages/recipe/index.js

```diff
@@ -810,10 +817,22 @@ class RecipePage extends React.Component {
             <p>Errata</p>
           </Tab>
         </Tabs>
-        <CreateComposition
-          recipe={this.state.recipe.name}
-          setNotifications={this.setNotifications}
-        />
+        {this.state.modalCreateComp ?
+          <CreateComposition
+            recipe={this.state.recipe.name}
+            setNotifications={this.setNotifications}
+            handleHideModalCreateComp={this.handleHideModalCreateComp}
```

Comment:
handleHideModalCreateComp function should be defined, like handleHideModalExport